docs/guides/00-integration-carto-vl.md

```diff
@@ -0,0 +1,165 @@
+<!-- Include Icons -->
+<link rel="stylesheet" href="https://libs.cartocdn.com/airship-icons/<VERSION>/icons.css">
+<!-- Include Web Components -->
+<script src="https://libs.cartocdn.com/airship-components/<VERSION>/airship.js"></script>
+```
+
+## Including CARTO-VL
+
+As pointed in the [CARTO-VL docs](https://carto.com/developers/carto-vl/guides/getting-started/) we can also include CARTO-VL in our app using a CDN
+
+```html
+<head>
+  <!-- Include CARTO VL JS -->
+  <script src="https://libs.cartocdn.com/carto-vl/v0.8.0/carto-vl.min.js"></script>
+  <!-- Include Mapbox GL JS -->
+  <script src="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.js"></script>
+  <!-- Include Mapbox GL CSS -->
+  <link href="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.css" rel="stylesheet" />
+</head>
+```
+
+
+## Setting a basic layout
+
+A basic layout will just include a container for our map and a sidebar for our future widget.
+
+An airship app always has a `main` tag with the `as-app-content` class.
+
+In this case we add a `as-map-wrapper` where the map will be displayed and a `as-sidebar` to contain our widgets.
+
+ ```html
+<main class="as-app-content">
+    <div class="as-map-wrapper">
+      <div id="map"></div>
+    </div>
+
+    <aside class="as-sidebar as-sidebar--right">
+
+    </aside>
+  </main>
+```
+
+
+## Drawing a map with CARTO-VL
+For this guide we are going to use the js code of the [CARTO-VL basic example](https://carto.com/developers/carto-vl/examples/#example-add-carto-dataset-layer at this point you should see a map with an white space on the right.
+
+```html
+<script>
+    const map = new mapboxgl.Map({
+      container: 'map',
+      style: 'https://basemaps.cartocdn.com/gl/voyager-gl-style/style.json',
+      center: [0, 30],
+      zoom: 2,
+      scrollZoom: false,
+      dragRotate: false,
+      touchZoomRotate: false
+    });
+
+    const nav = new mapboxgl.NavigationControl({
+      showCompass: false
+    });
+    map.addControl(nav, 'top-left');
+
+    // Define user
+    carto.setDefaultAuth({
+      user: 'cartovl',
+      apiKey: 'default_public'
+    });
+
+    // Define layer
+    const source = new carto.source.Dataset('ne_10m_populated_places_simple');
+    const viz = new carto.Viz();
+    const layer = new carto.Layer('layer', source, viz);
+
+    layer.addTo(map, 'watername_ocean');
+    layer.on('loaded', hideLoader);
+
+    function hideLoader() {
+      document.getElementById('loader').style.opacity = '0';
+    }
+</script>
+```
+
+## Adding a Category Widget
+
+Adding a Category Widget is as simple as including `<as-category-widget>` tag within our `as-sidebar` element, setting the options you prefer.
+
+ ```html
+<aside class="as-sidebar as-sidebar--right">
+  <as-category-widget
+    id="cities-widget"
+    heading="Populated places"
+    description="Sum of the population grouped by country">
```

Comment:
You need to change this description, as it was the previously one I used in my widget before changing widget data.